src/index.js

```diff
@@ -0,0 +1,58 @@
+import './index.css';
+// import { loadShows } from './modules/renderWindow.js';
+
+// loadShows();
```

Comment:
- [OPTIONAL] Kindly do not comment code. They can make your code look messy and do not add functionality to it.